src/components/Explore/index.js

```diff
@@ -0,0 +1,7 @@
+export { Explore } from "./Explore";
+export { Header } from "../Header";
+export { Footer } from "../Footer";
+export { Main } from "./Main";
+export { NavBar } from "../NavBar";
+export { VideoCard } from "./VideoCard";
+export { Filters } from "./Filters";
```

Comment:
I was importing first and then exporting, but there is no need to import first, I learnt this , now I can implement this in my project